lib/rucio/core/scope.py

```diff
@@ -64,12 +66,18 @@ def add_scope(scope, account, *, session: "Session"):
             raise Duplicate('Scope \'%s\' already exists!' % scope)
         else:
             raise RucioException(e)
-    except:
+    except Exception:
         raise RucioException(str(format_exc()))
 
 
 @read_session
-def bulk_add_scopes(scopes, account, skipExisting=False, *, session: "Session"):
+def bulk_add_scopes(
+    scopes: list[InternalScope],
+    account: InternalAccount,
+    skipExisting: Optional[bool] = False,
```

Comment:
```suggestion
    skipExisting: bool = False,
```

The `Optional` in `Optional[bool]` refers to the type of `skipExisting`, i.e. it could either be `None` or `bool`. In this context, we probably always expect this to be a `bool`, even when this parameter is not passed in a call to `bulk_add_scopes` (in which case it defaults to `False`).


Off-topic: `skipExisting` should be renamed to `skip_existing` for consistency with our naming convention